daemon.lua

```diff
@@ -20,7 +22,14 @@ function telegraf:push(message)
 	self:sendto(message, inet.localhost, 8094)
 end
 
-local inbox = mailbox.inbox(100 * 1024)
+local inbox = {
+	telegraf = mailbox.inbox(100 * 1024),
```

Comment:
I think `telegraf` is a bit misleading here.. I would name this `notify` instead..